flashinfer_trace/tests/references/test_gqa_ragged_prefill_h16_kv2_d128.py

```
@@ -0,0 +1,172 @@
+
+
+def main():
+    print("Testing GQA Ragged Prefill h16/kv2 (Qwen3-32B TP=4)")
+
+    test_configs = [(1, 16, 32), (4, 32, 64), (8, 64, 128)]
+    passed = 0
+    for batch_size, max_q_len, max_kv_len in test_configs:
+        try:
+            if test_correctness(batch_size, max_q_len, max_kv_len):
+                passed += 1
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)